files/image_config/interfaces/interfaces.j2

```diff
@@ -76,7 +76,7 @@ auto {{ member }}
 allow-hotplug {{ member }}
 iface {{ member }} inet manual
     pre-up teamdctl {{ pc }} port add {{ member }} || true
-    post-up ifconfig {{ member }} up
+    post-up ip link show {{ pc }} && ifconfig {{ member }} up
```

Comment:
you said that teamdctl port add will add the member while bring up the member, then why post-up is neccessary?